packages/react/accordion/src/Accordion.tsx

```diff
@@ -11,6 +11,9 @@ import { useId } from '@radix-ui/react-id';
 
 import type * as Radix from '@radix-ui/react-primitive';
 import type { Scope } from '@radix-ui/react-context';
+import { useDirection } from '@radix-ui/react-direction';
```

Comment:
We'll need to add this package as a dependency in accordion now and run `yarn` so it gets added to the lockfile.